src/dreyfus_index_updater.erl

```diff
@@ -101,14 +100,16 @@ purge_index(Db, IndexPid, Index) ->
                         true -> Acc;
                         false ->
                             update_or_delete_index(IndexPid, Db, DI, Proc),
-                            update_local_doc(Db, Index, PurgeSeq),
                             [{Id, Rev} | Acc]
                     end
             end,
-            clouseau_rpc:set_purge_seq(IndexPid, PurgeSeq),
-            {ok, Acc0}
+            [Acc0, PurgeSeq]
```

Comment:
did you want to use a tuple here: `{Acc0, PurgeSeq}? I have not see a list to be used like this